src/services/db.js

```diff
@@ -9,136 +10,153 @@ export default class Database {
 
     this.db = new Dexie("db");
 
-    this.db.version(1).stores({
-      // this is the table for the "predefined" tricks, the id's will start from 1000 onwards
-      mainTricks: "++id, alias, technicalName, establishedBy, yearEstablished, linkToVideo, startPos, endPos, difficultyLevel, description, tips",
-      mainAttributes: "id, stickFrequency",
-      userTricks: "++id, alias, technicalName, establishedBy, yearEstablished, linkToVideo, startPos, endPos, difficultyLevel, description, tips, strickfrequency",
-      combos: "++id, name, tricks, minDiff, maxDiff, avgDiff, totalDiff, numberOfTricks, establishedBy, yearEstablished, linkToVideo, comments, stickFrequency"
+    this.db.version(2).stores({
+      // to keep track of all the versions
+      versions: "key,version",
+      // this is the table for the "predefined" tricks, the id's will start from 10000 onwards
+      predefinedTricks: "id,alias,technicalName,establishedBy,yearEstablished,linkToVideo,videoStartTime,videoEndTime,startPos,endPos,difficultyLevel,description,tips,stickFrequency",
+      userTricks: "++id,alias,technicalName,establishedBy,yearEstablished,linkToVideo,videoStartTime,videoEndTime,startPos,endPos,difficultyLevel,description,tips,stickFrequency,deleted",
+      predefinedCombos: "id, name, *tricks, minDiff, maxDiff, avgDiff, totalDiff, numberOfTricks, establishedBy, yearEstablished, linkToVideo, comments, stickFrequency",
+      userCombos: "++id, name, *tricks, minDiff, maxDiff, avgDiff, totalDiff, numberOfTricks, establishedBy, yearEstablished, linkToVideo, comments, stickFrequency, deleted"
```

Comment:
Should we add startPos and endPos on the combos as well?
We could use it e.g. for combining combos in the future.

Reply:
In the combos we can also add videoStartTime and videoEndTime.

Reply:
both we can easily add in the future. I would focus now in this pull request more on getting the combos working nicely. see #115